client/transaction-pool/src/revalidation.rs

```diff
@@ -34,10 +34,15 @@ pub const BACKGROUND_REVALIDATION_INTERVAL: Duration = Duration::from_millis(5);
 
 const BACKGROUND_REVALIDATION_BATCH_SIZE: usize = 20;
 
-/// Payload from queue to worker.
-struct WorkerPayload<Api: ChainApi> {
-	at: NumberFor<Api>,
-	transactions: Vec<ExHash<Api>>,
+/// Background worker notification.
+enum WorkerNotify<Api: ChainApi> {
+	NewPayload {
+		at: NumberFor<Api>,
+		transactions: Vec<ExHash<Api>>,
+	},
+	Remove {
```

Comment:
```suggestion
	RemoveFromRevalidation {
```
?

Reply:
It does only revalidation, so it is redundant to me

Reply:
Okay, so let's just do `New/Remove` instead of `NewPayload/Remove`.

Reply:
Changed to Add/Remove